sunpy/net/tests/test_attr.py

```diff
@@ -329,6 +329,12 @@ def test_asterisk_attrs(ALL):
     assert "Instrument(all: All values of this type are supported.)" in repr(Instrument.all)
 
 
+def test_single_pair_argument_attrs(ALL):
+    # This checks we can submit * to mean all attrs.
+    with pytest.raises(ValueError):
+        attr.Attr.update_values({GenericClient: {Instrument: [("not star")]}})
```

Comment:
ALso what happens if I accidentally registered "*woops"? Does this pass or error?

Reply:
It shouldn't but I will add a test case